tests/unit/top-step-form.spec.ts

```diff
@@ -74,6 +76,39 @@ describe('Top Step Form', () => {
     });
   });
 
+  it('should accept templatable values', async () => {
```

Comment:
Should we add this kind of test in every step test files accepting templating ?

Reply:
Maybe but I think all steps (except the `Top` one) already accept strings so there's nothing much to test.

Reply:
Oook understood